terragraph_planner/data/template.yaml

```diff
@@ -1,106 +1,104 @@
-# This file is a template for the config yaml file with the default values used
-# in the planner, except all the file paths and DEVICE_LIST.
-# Change the values to customize the config of your own plan.
+# This file is a template for the configuration yaml file with the default
+# values used in the planner, except all the file paths and DEVICE_LIST. Change
+# the values to customize the configuration of your own plan.
 #
 # NOTE:
-# 1. The default value of each file path is an empty string rather than the string
-#    in this template.
-# 2. If any field is deleted, left blank or filled out with "null" or "~",
-#    the default value is used, except BOUNDARY_POLYGON_FILE_PATH and DEVICE_LIST.
-# 3. BOUNDARY_POLYGON_FILE_PATH must have valid values when los computation is ran.
-# 4. DEVICE_LIST must have valid values.
-# 5. The "[Optional]" tag for each field means it can be null and its default Python
-#    value is None. Other fields without that tag cannot be assigned as null.
-# 6. It's not recommended to leave a field blank.
-# 7. "None" is treated as a string "None" rather than None type in Python. Please
-#    refer to https://yaml.org/ for more information.
+# 1. BOUNDARY_POLYGON_FILE_PATH is a required field when LOS analysis is run.
+# 2. The DEVICE_LIST cannot be empty.
+# 3. If any field is deleted, left blank or filled out with "null" or "~", the
+#    default value is used. This does not apply to BOUNDARY_POLYGON_FILE_PATH
+#    and DEVICE_LIST.
+# 4. The default value of each file path is null rather than the string in this
+#    template.
+# 5. The "[Optional]" tag for each field means it can be null and its default
+#    Python value is None. Other fields without that tag cannot be assigned as
+#    null or omitted entirely.
+# 6. It is not recommended to include a field but leave its value blank. It is
+#    better to omit it entirely or fill it out with "null" or "~".
+# 7. "None" is treated as a string "None" rather than None type in Python.
+#    Please refer to https://yaml.org/ for more information.
+#
+# Because of how defaults are handled, the input configuration yaml file only
+# needs to provide the parameters that are different than the given default.
+#
 
 SYSTEM:
   # [String] The directory to dump all the output and debug files. The output
-  # files will get dumped under a subfolder "output" and the debug files will get
-  # dumped undr a subfolder "debug". If they do not exist, the planner will create
-  # them.
+  # files are stored under the folder "output" and the debug files, if
```

Comment:
Add a comma after "output" (and I think grammatically in English commas go inside the quotation marks but in technical discussions like this I usually leave them outside in order to be precise as to what actually is inside the quotation marks).

Reply:
I'm not so sure there should be a comma. The A are this and the B are that. I don't think that sentence should have a comma in it.